go.mod

```diff
@@ -159,3 +162,7 @@ replace github.com/golangci/golangci-lint => github.com/golangci/golangci-lint v
 replace github.com/filecoin-project/filecoin-ffi => ./extern/filecoin-ffi
 
 replace github.com/filecoin-project/test-vectors => ./extern/test-vectors
+
+replace github.com/filecoin-project/go-fil-markets => ../go-fil-markets
+
+replace github.com/filecoin-project/go-data-transfer => ../go-data-transfer
```

Comment:
Remember to update these!

Reply:
We need to test these first, so no way around not replacing `go-data-transfer` to the one from my branch, as the `go-data-transfer` PR is not merged (same as `go-fil-markets`).

I guess once this is tested we first merge `go-fil-markets`, then `go-data-transfer`, issue releases, update `go.mod` here, and then merge this.